### DISCUSSION TITLE
Error: connector is required

### DISCUSSION THREAD

**Discussion by BakhtiyarKZ** [wevm/wagmi#683] [General]
Hello, after updating from 0.3 to 0.5 I have this error. How I can resolve it?
`Error: connector is required
    at Object.mutationFn$5 [as mutationFn] `
![weesd](https://user-images.githubusercontent.com/48028890/177982857-ffec51a0-7d62-4971-9f67-c280a9e19506.PNG)

**Comment by nasa8x** [comment]
There is a slight change in version 0.5 you can see an example here: <https://wagmi.sh/docs/hooks/useConnect>

```js 
const { connect, connectors } = useConnect();
```
```html
<button onClick={() => connect({ connector: connectors[0] })}>
        Connect
</button>
```

**Reply by BakhtiyarKZ** [reply]
Thank you very much, it works
